content/ft_ensina/load_content.py

```diff
@@ -0,0 +1,21 @@
+from markdown2 import markdown
+from lxml import html
+import os
+
+def load_ft_ensina_videos(dict):
+    video_dir = 'content/ft_ensina/videos/'
+    filenames = [x for x in sorted(os.listdir(video_dir)) if x.endswith('.md')]
+    content = [html.fromstring(markdown(open(video_dir+x, 'r').read())) for x in filenames]
```

Comment:
Deu uma boa melhorada, mas eu aconselharia a trocar esse tanto de `for` por apenas 1 mais ou menos no esquema de baixo:
```python3
for item in filenames:
    content = html.fromstring(markdown(open(video_dir + item, 'r').read()))
    all_data[item] = {
        "disciplina": content.xpath('//h4/text()'),
        "títulos": content.xpath('//h4/text()'),
        "assuntos": content.xpath('//h4/text()'),
        "links": content.xpath('//h1/text()'),
    }
```

Reply:
Bem mais enxuto!